app/managers.py

```diff
@@ -0,0 +1,41 @@
+import sqlite3
+
+from models import Actor
+
+
+class ActorManager:
+
+    def __init__(self) -> None:
+        self._connection = sqlite3.connect("cinema.db")
+        self.data_base = "cinemas"
```

Comment:
The table name is set to `cinemas`, which is incorrect according to the task requirements. Update `self.data_base` to `actors` to match the task requirements and ensure the database operations target the correct table.